pkg/apply/taskrunner/runner_test.go

```diff
@@ -244,18 +248,13 @@ func TestBaseRunner(t *testing.T) {
 			close(eventChannel)
 			wg.Wait()
 
-			if tc.expectedError != nil {
-				assert.IsType(t, tc.expectedError, err)
-				if timeoutError, ok := err.(*TimeoutError); ok {
-					assert.ElementsMatch(t, tc.expectedTimedOutResources,
-						timeoutError.TimedOutResources)
-					assert.Equal(t, timeoutError.Error(), tc.expectedErrorMsg)
-				}
-				return
-			} else if err != nil {
+			if err != nil {
 				t.Errorf("expected no error, but got %v", err)
 			}
 
+			for _, event := range events {
```

Comment:
Not sure if you meant to leave this in here. Looks like debugging.
I plan to go through and convert unit tests to test the full event log with the new methods I used for the e2e tests. I can probably clean this up later.